example/lib/main.dart

```diff
@@ -220,7 +224,11 @@ class _MyAppState extends State<MyApp> {
           value: language,
           items: getLanguageDropDownMenuItems(),
           onChanged: changedLanguageDropDownItem,
-        )
+        ),
+        Visibility(
+          visible: Platform.isAndroid,
```

Comment:
Unfortunately web doesn't support this yet, so you'll need to set this to `!kIsWeb && Platform.isAndroid`.

Reply:
Another option here is to add this.
```
  bool get isIOS => !kIsWeb && Platform.isIOS;

  bool get isAndroid => !kIsWeb && Platform.isAndroid;

  bool get isWeb => kIsWeb
```

And we can reference those, otherwise feel free to just add the `!kIsWeb && Platform.isAndroid` check and I can refactor it later.